### ISSUE TITLE
street_address method returning nil

### ISSUE THREAD

**Issue by DTwigs** [AuthorizeNet/sdk-ruby#51]
The street_address method is returning nil.

What's weird is that when I use the deprecated call `Cim.new.get_customer_profile(user.acid)` from the AuthNetApi the response has the address in it.

```
resp = cim.get_profile(user.acid)
resp.profile.payment_profiles.billing_address.street_address #=> nil
```

Have any idea why this is happening?

**Comment by brianmc** [comment]
Looking at this one @DTwigs, we'll update the sample at https://github.com/AuthorizeNet/sample-code-ruby/blob/master/CustomerProfiles/get-customer-profile.rb as soon as we can and of course update the SDK if we find the issue there.

**Comment by brianmc** [comment]
Fixed in the last release 1.8.7
